python/common/org/python/types/Set.java

```diff
@@ -430,24 +430,41 @@ public boolean __setattr_null(java.lang.String name, org.python.Object value) {
     }
 
     @org.python.Method(
-            __doc__ = ""
+            __doc__ = "check whether the given two sets have no elements in common",
+            args = {"other"}
     )
     public org.python.Object isdisjoint(org.python.Object other) {
-        throw new org.python.exceptions.NotImplementedError("isdisjoint() has not been implemented.");
+        java.util.Set set = ((Set) this.copy()).value;
+        set.retainAll(((Set) other).value);
+        return new org.python.types.Bool(set.isEmpty());
     }
 
     @org.python.Method(
-            __doc__ = ""
+            __doc__ = "state whether other set is a subset",
+            args = {"other"}
     )
     public org.python.Object issubset(org.python.Object other) {
-        throw new org.python.exceptions.NotImplementedError("issubset() has not been implemented.");
+        if (other instanceof org.python.types.Set) {
+            org.python.types.Set otherSet = (org.python.types.Set) other;
+            return new org.python.types.Bool(otherSet.value.containsAll(this.value));
+        } else if (other instanceof org.python.types.Int) {
+            throw new org.python.exceptions.TypeError("'int' object is not iterable");
```

Comment:
It seems quite unlikely that `issubset()` and `issuperset()` only throw a TypeError for int objects. Per the [python stdtypes docs](https://docs.python.org/3/library/stdtypes.html#set): 
> Note, the non-operator versions of union(), intersection(), difference(), and symmetric_difference(), issubset(), and issuperset() methods will accept any iterable as an argument.

Perhaps the logic here (and in `issuperset()`) should handling sets/frozensets, then attempting to call `org.Python.iter(other)`, which either throws the appropriate TypeError or returns the Iterable, which is then used to perform the `issubset()` check.

All these should be exercised in the test cases. Also, if there's any case where NotImplemented is returned, that should be tested as well.